make_release.sh

```diff
@@ -58,19 +54,22 @@ cd ..
 
 mv domjudge domjudge-$VERSION
 
-tar -cf "domjudge-$VERSION.tar" "domjudge-$VERSION"
-gzip -9 "domjudge-$VERSION.tar"
+tar -cf "domjudge-$TAG.tar" "domjudge-$VERSION"
+gzip -9 "domjudge-$TAG.tar"
```

Comment:
Nitpick: since `$TAG` is now used, it would be nice to check that `$TAG` and `$VERSION` match (maybe up to case for RC releases).

Reply:
It did in the past, but to make it possible to do a testrun via GitLab (where this is not the case yet) this should not hold. The behaviour you describe should be implemented in lines 42-45 where my intention is to break when this does not hold..

Or do you refer to the fact that one could publish a tag A.B.0 but is actually at VERSION: C.D.E?

Reply:
The latter indeed: make sure that if a tag is specified, then it should match the version.

Not sure I understand your first case. Are you refering to when there is no git tag yet?

Reply:
Yes, see https://gitlab.com/mvasseur/domjudge-packaging/-/pipelines/274044040

This way you can already build a (testing unreleased) tarball before, possibly run some tests on it (I can see different usecases) and also share how to test these things. The advantage of running this on GitLab is that you can test on a clean environment, but at this stage the tag is not pushed yet and you would use the branch.